src/imitation/zero_one_loss.jl

```diff
@@ -0,0 +1,58 @@
+"""
```

Comment:
Maybe we should merge this file with `ssvm_loss.jl`

Reply:
To me this file shows an implementation of ImitationLoss and SSVMLoss at the same time so it makes sense to keep it separate.